docs/migration-guides/v4/gux-action-button-legacy.md

```diff
@@ -0,0 +1,40 @@
+# gux-action-button-legacy
+
+[Back to main guide](./readme.md)
+
+## V3 Example
+
+```html
+<gux-action-button-legacy text="Primary" accent="primary" is-open="true">
+  <gux-action-item text="test"></gux-action-item>
+  <gux-action-item text="test2"></gux-action-item>
+  <gux-action-item text="test3"></gux-action-item>
+  <gux-list-divider></gux-list-divider>
+  <gux-action-item><span>I am a span</span></gux-action-item>
+</gux-action-button-legacy>
+```
+
+## V4 Migration
+
+Steps:
+
+- Replace the `gux-action-item` tag name with `gux-list-item`
+  - The `gux-list-item` does not have a `text` property. All content must be slotted within `gux-list-item`.
+  - The `gux-list-item` does not have a `value` property so this information will have to be passed to your `click` handler directly
+
+```diff
+-  <gux-action-item text="test"></gux-action-item>
++  <gux-list-item>test</gux-list-item>
+```
+
+Completed V3 Full Migration:
+
+```html
+  <gux-action-button text="Primary" accent="primary" is-open="true">
```

Comment:
I think the `text` prop is no longer valid here and has been replaced with a title `slot`

Reply:
Correct, I am going to close this PR and this is fixed in https://github.com/MyPureCloud/genesys-spark/pull/149